paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -1316,22 +1297,17 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
 
     @Test
     fun whenVerticalModeScreen_becomesCurrentScreen_updateSelectionCalled() {
-        var updatedSelection: PaymentSelection? = null
-        fun onUpdateSelection(paymentSelection: PaymentSelection?) {
-            updatedSelection = paymentSelection
-        }
-
         val verticalModeSelection = PaymentSelection.GooglePay
 
         runScenario(
             initialSelection = verticalModeSelection,
             initialIsCurrentScreen = false,
             formTypeForCode = { FormHelper.FormType.Empty },
-            updateSelection = ::onUpdateSelection,
         ) {
             isCurrentScreenSource.value = true
 
-            assertThat(updatedSelection).isEqualTo(verticalModeSelection)
+            assertThat(selection.value).isEqualTo(verticalModeSelection)
+            assertThat(updateSelectionTurbine.awaitItem()).isEqualTo(Unit)
```

Comment:
```suggestion
            assertThat(updateSelectionTurbine.awaitItem()).isTrue()/.isFalse()
```

Which would validate we're calling this with the right params.